packages/next/client/image.tsx

```diff
@@ -291,6 +325,14 @@ export default function Image({
   const heightInt = getInt(height)
   const qualityInt = getInt(quality)
 
+  const MIN_IMG_SIZE_FOR_PLACEHOLDER = 5000
+  const tooSmallForBlurryPlaceholder =
+    widthInt && heightInt && widthInt * heightInt < MIN_IMG_SIZE_FOR_PLACEHOLDER
+  const shouldShowBlurryPlaceholder =
+    configEnableBlurryPlaceholder &&
+    placeholder === 'blur' &&
+    !tooSmallForBlurryPlaceholder
```

Comment:
```suggestion
  const shouldShowBlurryPlaceholder =
    placeholder === 'blur' &&
    !tooSmallForBlurryPlaceholder
```

No longer need to check config

Reply:
Thanks for the catch, done.